turbolinks/checkout.html

```diff
@@ -0,0 +1,311 @@
+<!doctype html>
+	<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="">
+		<meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+		<meta name="generator" content="Hugo 0.84.0">
+		<title>PixieBrix Turbolinks</title>
+
+		<link rel="canonical" href="https://getbootstrap.com/docs/5.0/examples/dashboard/">
+
+		<!-- Bootstrap core CSS -->
+		<link href="./assets/dist/css/bootstrap.min.css" rel="stylesheet" data-turbolinks-track="reload">
+		<meta name="turbolinks-visit-control" content="reload">
+
+		<style>
+		.bd-placeholder-img {
+			font-size: 1.125rem;
+			text-anchor: middle;
+			-webkit-user-select: none;
+			-moz-user-select: none;
+			user-select: none;
+		}
+
+		@media (min-width: 768px) {
+			.bd-placeholder-img-lg {
+				font-size: 3.5rem;
+			}
+		}
+	</style>
+
+
+	<!-- Custom styles for this template -->
+	<link href="./dashboard.css" rel="stylesheet" data-turbolinks-track="reload">
```

Comment:
> I think we need to apply the same vercel.json fix that we did for the previous projects? The URL link on the main page doesn't work because of "/" issues?

With turbolinks the pages are real, they aren't made up via JS in index.html. The issue here is that `./` urls point to root when opening a link without trailing slash:

```
https://playground-11eu1g2m8-pixiebrix.vercel.app/turbolinks
```
They only work when appending it:

```
https://playground-11eu1g2m8-pixiebrix.vercel.app/turbolinks/
```

Since the vercel index does not append slashes, we can change these links to
```suggestion
	<link href="/turbolinks/dashboard.css" rel="stylesheet" data-turbolinks-track="reload">
```

Alternatively I think adding this single tag in the headerwill work:

```html
<base href="/turbolinks/">
```